pom.xml

```diff
@@ -51,18 +61,13 @@
     </dependency>
     <dependency>
       <groupId>${project.groupId}</groupId>
-      <artifactId>jitsi-protocol-jabber</artifactId>
-      <version>2.9-20151214.211356-5</version>
-    </dependency>
-    <dependency>
-      <groupId>${project.groupId}</groupId>
       <artifactId>jitsi-ui-service</artifactId>
-      <version>2.9-20150723.181654-1</version>
+      <version>${jitsi-desktop.version}</version>
```

Comment:
I've noticed that some tooling works better when not using property for version numbers. Granted, that's more of a bug in that tooling, but if all you're doing is replacing two identical version strings with a property value, it might be worth while to simply use duplicate values here. This argument is not very strong, and should be ignored if you use `${jitsi-desktop.version}` somewhere else too.

Reply:
It's rather important that all Jitsi Desktop dependencies are at the same version. If I explicitly state the version, this will definitely get out of sync again. Jicofo only has two dependencies, but have a look at Jigasi: there are almost all Maven packaged submodules of Jitsi Desktop referenced and there was (is in master) 2.9-SNAPSHOT mixed with 2.11-SNAPSHOT. It's pure luck this was (is) working at all.

Reply:
Well... using -SNAPSHOT defeats the purpose a bit there, but I get what your'e saying. :)

Reply:
The -SNAPSHOT obviously isn't going to stay.